src/__tests__/a11yAPI.test.tsx

```diff
@@ -220,14 +219,15 @@ test.skip('getByA11yStates, queryByA11yStates', () => {
     'disabled',
   ]);
   const disabled = queryByA11yStates(['disabled']);
-  expect(disabled && disabled.props.accessibilityStates).toMatchObject([
+  expect(disabled?.props.accessibilityStates).toMatchObject([
     'selected',
     'disabled',
   ]);
   const disabledSelected = queryByA11yStates(['selected', 'disabled']);
-  expect(
-    disabledSelected && disabledSelected.props.accessibilityStates
-  ).toEqual(['selected', 'disabled']);
+  expect(disabledSelected?.props.accessibilityStates).toEqual([
```

Comment:
It runs TS linter rules now, is it something we want? Perhaps we need to make sure that this doesn't have more impact (some JS rules disappearing, or something like that?)

Reply:
Since it extends `callstack` ESLint preset I think it's more of a concern of `@callstack/eslint-config` to maintain parity between the two? I've quickly checked and I don't see glaring differences between JS/TS configs. @thymikee do you think it may be a problem?